server/pension/tests.py

```diff
@@ -1,3 +1,8 @@
 from django.test import TestCase
 
-# Create your tests here.
+class TestPension(TestCase):
+    def test_index_view(self):
+        response = self.client.get('/index/')
```

Comment:
@ranmoshe 're-roll' means go to master branch, pull the latest changes, come back to your branch, do a merge from master to your branch (to get the latest changes from master in your branch). PR is Pull Request.